frontend/src/app/app.module.ts

```diff
@@ -54,7 +55,8 @@ import { LeadService } from './services/lead.service';
     MatSidenavModule,
     MatToolbarModule,
     MatTableModule,
-    MatPaginatorModule
+    MatPaginatorModule,
+    MatSortModule
   ],
   providers: [
     LeadService,
```

Comment:
Remove the LeadService provider here since it's already provided at the root level.

Reply:
Ight